packages/react-dev-utils/noopServiceWorkerMiddleware.js

```diff
@@ -7,9 +7,11 @@
 
 'use strict';
 
-module.exports = function createNoopServiceWorkerMiddleware() {
+const path = require('path');
+
+module.exports = function createNoopServiceWorkerMiddleware(servedPath = '/') {
   return function noopServiceWorkerMiddleware(req, res, next) {
-    if (req.url === '/service-worker.js') {
+    if (req.url === path.join(servedPath, 'service-worker.js')) {
```

Comment:
@dliv-shipt I tested this locally and it doesn't appear that this fixes the issue. I believe the reason behind it is because we aren't actually using this version of `react-dev-utils` because this is our forked version. Instead, it is using the _real_ `react-dev-tools` so this change doesn't affect anything. I took a screenshot of `react-dev-tools` in the `node_modules` folder and you can see that `servedPath` doesn't have a default like I added above:

![image](https://user-images.githubusercontent.com/34040106/109308999-5705c700-7808-11eb-9d1c-323b10a5b4d4.png)

Reply:
Yes, you can back that out. I was wondering if that might be what was breaking the scripts at build time but after looking closer I think what you have now is right (updating the `paths.js` file and stubbing files at `packages/cra-template/template/src/{context,module}.js`).